### PR TITLE
Fix #146 Add deep-link support Announcement

### LINKED ISSUE #146: Add deeplink support
## Kind (Required)

- Improvement

 ## Overview (Required)

- We should add a deep link to here for the announcement screen.
- I want to use the navigation component deep link support
- Probably you can check it like this

```
adb shell am start -a android.intent.action.VIEW -d "https://droidkaigi.jp/2019/announcement"
```

### PR DESCRIPTION
**Pull request by KoutaMatsushita**
## Issue
- close #146

## Overview (Required)
- Add DeepLinks Setting

## Links
- https://medium.com/@star_zero/navigation%E3%81%AEimplicit-deep-link%E8%A9%B3%E7%B4%B0-1e2d955f41ae

## Screenshot
None

### CODE REVIEW

**Comment by jmatsu** [comment]
👀

File: corecomponent/androidcomponent/src/main/res/navigation/navigation.xml [hunk 1]
```diff
@@ -140,8 +140,11 @@
     <fragment
         android:id="@+id/announcement"
         android:name="io.github.droidkaigi.confsched2020.announcement.ui.AnnouncementFragment"
-        android:label="@string/announce_label"
-        />
+        android:label="@string/announce_label">
+        <deepLink
+            android:id="@+id/deepLink"
+            app:uri="https://droidkaigi.jp/2019/announcement" />
```
Comment (jmatsu) [inline corecomponent/androidcomponent/src/main/res/navigation/navigation.xml]: 2020's URL will be the following. 

```suggestion
            app:uri="https://droidkaigi.jp/2020/announcement" />
```

Comment (KoutaMatsushita) [reply corecomponent/androidcomponent/src/main/res/navigation/navigation.xml]: careless mistake 💦 
thanks! 😄

**Comment by KoutaMatsushita** [comment]
@jmatsu 
I fixed.
check it again please 🙇

**Pull request merged**